infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
 		PromQL: `pd_region_syncer_status{$LABEL_CONDITIONS}`,
 		Labels: []string{"instance", "type"},
 	},
+	"tikv_engine_size": {
+		PromQL:  `sum(tikv_engine_size_bytes{$LABEL_CONDITIONS}) by (instance, type, db)`,
+		Labels:  []string{"instance", "type", "db"},
+		Comment: "The storage size per TiKV instance",
+	},
+	"tikv_store_size": {
+		PromQL:  `sum(tikv_store_size_bytes{$LABEL_CONDITIONS}) by (instance,type)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The available/capacity size of each TiKV instance",
```

Comment:
I don't understand. Is it available size or capacity?

Reply:
yes, I will change those comment, Great catch.